package.json

```diff
@@ -201,7 +202,9 @@
     "fix-css": "npm run lint-css --fix",
     "fix": "npm run fix-js && npm run fix-css",
     "test-file": "mocha",
+    "test-file-ci": "mocha --reporter mocha-junit-reporter",
```

Comment:
Add a `.gitignore` entry for `test-results.xml` in case someone mistakenly runs this locally.